### DISCUSSION TITLE
ФРМО. Описание территориального признака. Есть ли в этом информация?

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#68] [Административная РГ]
строка 164 описание территориального признака (указывается только для государственных МО)

Выглядит, как вычисляемый из подчиненности. 

Кажется, что просто для статистики «сколько у нас районных, а сколько межрайонных» держать эту информацию бессмысленно.

**Comment by IvanekOS** [comment]
> строка 164 описание территориального признака (указывается только для государственных МО)
> 
> Выглядит, как вычисляемый из подчиненности.
> 
> Кажется, что просто для статистики «сколько у нас районных, а сколько межрайонных» держать эту информацию бессмысленно.

Добрый день!
Я нашел единственное упоминание, связанное с территориальным признаком в нормативном пространстве - Приказ Минздрава России от 06.08.2013 N 529н (ред. от 19.02.2020) "Об утверждении номенклатуры медицинских организаций", II раздел. Предполагаю, что из-за этого приказа это поле и требуется хранить в явном виде.

**Reply by iradio** [reply]
Если требуется хранить в ФРМО, но можно вычислить из адреса или подчиненности - такой атрибут не должен включаться в профиль.
